src/mrnet/network/reaction_network.py

```diff
@@ -1594,6 +1632,162 @@ def mols_w_cuttoff(RN_pr_solved, cutoff=0, build_pruned_network=True):
         else:
             return mols_to_keep, pruned_entries_list
 
+
+    @staticmethod
+    def generate_node_string(combined_reactants, combined_products):
+        """
+        Method generate node string based on reactants and products
+        :param combined_reactants: list of node numbers, ex [100, 30], max length of 2
+        :param combined_products: list of node numbers, ex [100, 30], max length of 2
+        """
+        node_str = None
+        if len(combined_reactants) <= 2 and len(combined_products) <= 2:
+            if len(combined_reactants) == 2 and len(combined_products) == 2:
+                node_str = str(combined_reactants[0]) + "+" + "PR_" + str(combined_reactants[
+                                                                              1]) + "," + str(
+                    combined_products[0]) + "+" + str(
+                    combined_products[1])
+            elif len(combined_reactants) == 2 and len(combined_products) == 1:
+                node_str = str(combined_reactants[0]) + "+" + "PR_" + str(combined_reactants[
+                                                                        1]) + "," + str(
+                    combined_products[0])
+            elif len(combined_reactants) == 1 and len(combined_products) == 2:
+                node_str = str(combined_reactants[0]) + "," + str(
+                    combined_products[0]) + "+" + str(
+                    combined_products[1])
+            elif len(combined_reactants) == 1 and len(combined_products) == 1:
+                node_str = str(combined_reactants[0]) + "," + str(
+                    combined_products[0])
+            else:
+                node_str = None
+                print("Something is wrong", combined_reactants, combined_products)
```

Comment:
Don't know the answer, just wondering - is this something that's bad enough to merit an error?